### PR TITLE
Bundle Python for Builds and RPMs

### PR DESCRIPTION
**Pull request by tst-cjeffries**
Bundles python so that a we can rely on a specific version of python being available. Python for the build can now be built and installed via a shell script. Python is now built as part of the OpenGEE build, so that RPMs can be generated for distribution alongside the OpenGEE RPMs.

To test:
* Build and install opengee following the build instructions. Verify that OpenGEE works.
* Build and install the rpms, and verify that OpenGEE works.

### CODE REVIEW

File: earth_enterprise/rpms/build.gradle [hunk 1]
```diff
@@ -240,6 +242,91 @@ task stageOpenGeeInstall(type: Exec, dependsOn: 'compileOpenGee') {
         "installdir=${stagedInstallDir}")
 }
 
+// RPM that installs python 2 if needed
+task openGeePython2Rpm(type: GeeRpm) {
+    mustRunAfter stageOpenGeeInstall
+
+    packageName = 'python-2.7.18'
```
Comment (tst-rwildes) [inline earth_enterprise/rpms/build.gradle:249]: for the name should we call it gee-python2?

File: earth_enterprise/rpms/build.gradle [hunk 2]
```diff
@@ -240,6 +242,91 @@ task stageOpenGeeInstall(type: Exec, dependsOn: 'compileOpenGee') {
...
+    from (stagedInstallDir_python_2) {
+        into packageInstallRootDir
+    }
+}
+
+// RPM that installs python 3 if needed
+task openGeePython3Rpm(type: GeeRpm) {
+    mustRunAfter stageOpenGeeInstall
+
+    packageName = 'python-3.8.6'
```
Comment (tst-rwildes) [inline earth_enterprise/rpms/build.gradle:292]: same here for the name call it gee-python3

File: earth_enterprise/rpms/build.gradle [hunk 3]
```diff
@@ -541,16 +618,12 @@ task openGeeServerRpm(type: GeeRpm, dependsOn: openGeeServerInitScripts) {
...
-    // The WMS Service requires PIL, which is not automatically detected:
-    requires('python-imaging')
-    requires('python-lxml')
 
     requiresCommands(
         packageSharedCommands +
         [
             // Install script commands:
-            'cat', 'chmod', 'chown', 'cut', 'find', 'getent', 'mkdir', 'python',
+            'cat', 'chmod', 'chown', 'cut', 'find', 'getent', 'mkdir', 'python2.7',
```
Comment (tst-rwildes) [inline earth_enterprise/rpms/build.gradle:626]: Should we add gee-python2 and gee-python3?

Comment (tst-cjeffries) [reply earth_enterprise/rpms/build.gradle]: For now I don't think so. Our python rpm is more or less a 1-1 replacement of the system python packages that aren't available. Requiring just python (until we totally move to python 3) should be sufficient, and should cover both our python installs and the system ones if they user gets them.

**Review by tst-mswartz — APPROVED** [review]
Reviewed but not tested, looks good to me

**Review by tst-dsugar — APPROVED** [review]
This built ok for me

**Comment by tst-cjeffries** [comment]
Basil cannot cannot work out the CLA at the moment, however he has signed one.

**Pull request merged**